packages/welldone-wallet/src/lib/welldone.ts

```diff
@@ -361,9 +360,10 @@ export function setupWelldoneWallet({
   deprecated = false,
 }: WelldoneWalletParams = {}): WalletModuleFactory<InjectedWallet> {
   return async () => {
-    const isSupported =
-      !isMobile() && isSupportBrowser(["chrome", "edge-chromium", "opera"]);
-    if (!isSupported) {
+    const mobile = isMobile();
+    const supported = isBrowserSupported(["chrome", "edge-chromium", "opera"]);
```

Comment:
can we keep the name `isSupported` as it is boolean

Reply:
Alright will change it 👍